src/gtimelog/tests/core/test_time.py

```diff
@@ -152,6 +159,11 @@ def doctest_timewindow_last_entry():
 
 
     Case #4: several items
+    # FIX: datetime.datetime
+    # As python 3.7 and 3.8 doesn't send the same representation
+    # for duration : using seconds instead of __repr__
+    # python3.7 result : datetime.timedelta(0, 1860)
+    # python3.8 result : datetime.timedelta(seconds=1860)
```

Comment:
Oh.  Was the doctest checker doing more than just normalizing `u` prefixes on strings?

Reply:
Maybe... But in all case basing a test on this representation is not a good idea. More flexible and more importantly, basing test on real values (in this case : seconds, hours, minutes, whatever and surely not the representation of an object)

Reply:
Yeah, doctests have their drawbacks.  `assert last_entry.duration == datetime.timedelta(minutes=31)` would be much clearer.